docs/ssl_certs.md

```diff
@@ -2,16 +2,11 @@
 
 Empire allows you to attach IAM server certificates to an applications web process. Follow these steps to attach an SSL certificate.
 
-First, [upload a certificate to IAM](http://docs.aws.amazon.com/cli/latest/reference/iam/upload-server-certificate.html):
+Upload the certificate using the command below.
+The .pem file should contain a full chain of the public certificate and any root certificates.
 
-```console
-$ aws iam upload-server-certificate --server-certificate-name myServerCertificate --certificate-body file://public_key_cert_file.pem --private-key file://my_private_key.pem --certificate-chain file://my_certificate_chain_file.pem
 ```
-
-Then attach it to the application:
-
-```console
-$ emp certs-attach arn:aws:iam::<accountId>:server-certificate/myServerCertificate -a <app>
+emp ssl-cert-add -a <app> cert.pem cert.key
```

Comment:
Ah ok! Should we update the command list printed when running emp without any arguments? These methods are  currently listed:
```
   ssl                show ssl endpoint info
   ssl-cert-add       add a new ssl cert
   ssl-cert-rollback  add a new ssl cert
   ssl-destroy        destroy ssl endpoint
```